pkg/virt-api/webhooks/mutating-webhook/mutators/namespace-limits.go

```diff
@@ -44,10 +44,10 @@ func isResourceRequirementMissing(resourceRequirements kubev1.ResourceRequiremen
 	return memoryOrCpuIsMissing(resourceRequirements.Limits) || memoryOrCpuIsMissing(resourceRequirements.Requests)
 }
 
-func applyNamespaceLimitRangeValues(vmi *kubev1.VirtualMachineInstance, limitrangeInformer cache.SharedIndexInformer) {
+func applyNamespaceLimitRangeValues(vmi *kubev1.VirtualMachineInstance, limitrangeInformer cache.SharedIndexInformer, namespace string) {
```

Comment:
can't we take the namespace from *VMI?

Reply:
No, the VMI object can be missing the namespace when it's being created.

Reply:
@xpivarc Since vmi namespace, if present, must match the request namespace, it might not be correct to simply set `newVMI.Namespace = ar.Request.Namespace`? Thanks

Reply:
Right, but if we know the namespace from `ar.Request.Namespace` why can't we just set it in the VMI object?

Reply:
Well, I don't think we should do it. Let's consider the following case. The user has a VMI YAML with namespace `custom` but his `kubectl` is set to work with namespace `test`. What namespace is the intended namespace to be used? Further, this is how any other Kubernetes object would behave.

Reply:
I'm only referring to the case when there is no namespace set in the vmi object.

Reply:
What is the benefit of handling it if Kubernetes does it for us?

Reply:
I think if we don't, then it creates some kind of confusion.. at least for me. 

I guess what I don't understand is: if `vmi.Namespace` != `ar.Request.Namespace` why are we allowed to use the limits from `ar.Request.Namespace` ?

Reply:
The request will be denied in the end so I don't see harm here.

Reply:
@vladikr pushed test for this